sionna/mapping.py

```diff
@@ -980,6 +980,8 @@ def __init__(self,
                                               dtype.real_dtype,
                                               **kwargs)
 
+        self._no_threshold = np.finfo(dtype.as_numpy_dtype).tiny
```

Comment:
last comment: I would suggest that we cast this directly to a tensor
tf. cast(np.finfo(dtype.as_numpy_dtype).tiny,
            dtype.real_dtype)

Reply:
Done!